### DISCUSSION TITLE
[Request] r/EngineeringResumes Template

### DISCUSSION THREAD

**Discussion by TheColorDarkGreen** [subidit/rover-resume#1] [General]
Hi @subidit, could this template be added to rover-resume?

- https://www.reddit.com/r/EngineeringResumes/wiki/resumetemplates#wiki_latex
- https://www.overleaf.com/project/65a471c32f9ee1db3a96cc75
- https://imgur.com/NGtqtIl

**Comment by subidit** [comment]
> could this template be added to rover-resume?

I am not really sure what you mean by this. This is not a compilation repo of other templates. Anyway the resume you shared is a simple and good one. I'm assuming you want to create a similar looking résumé. You can use the Base Rover template for the same. In order to get the exact same formatting like [**Job Title,** Company - City, ST] in the same line- you could use the follwing command 

```latex
% Similar to EngineeringResumes
\subsection{Job Title, \normalfont Company - City, ST \hfill XXXX - Present}

% Even better
\subsection{Job Title $|$ \normalfont \textit{Company} - City, ST \hfill YYYY -- YY}

% Name header
\begin{center}
  {\Huge\bfseries BASIC ROVER} \par\medskip
  
  1 Infinite Loop, Menlo Park, CA $|$
  123 456 7890 $|$
  basic-rover@email.com
\end{center}
```

Rest all looks the same as Base Rover template (Base Rover has better bulleted list formatting).  Basically use `\normalfont` to remove the bold formatting inside the `\subsection{}`. Let me know if you have any other queries regarding this.
